guides/source/getting_started.md

```diff
@@ -1510,7 +1510,7 @@ comments on articles.
 
 We're going to see the same generator that we used before when creating
 the `Article` model. This time we'll create a `Comment` model to hold
-reference of article comments. Run this command in your terminal:
+reference to an article. Run this command in your terminal:
```

Comment:
I think it's fine to say reference here. For me the use of the word reference here isn't special (no backticks), but sets up the reference coming in migration nicely.